src/test/java/com/exasol/adapter/dialects/postgresql/CollectionTests.java

```diff
@@ -0,0 +1,17 @@
+package com.exasol.adapter.dialects.postgresql;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+class CollectionTests {
```

Comment:
Why do we need this test?

Reply:
Damn. That were a few experiments I did to verify that `containsAll` does what I expect. I forgot to delete them.

Reply:
Removed.